IdentityCore/src/MSIDConstants.m

```diff
@@ -21,8 +21,22 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
+#import "NSOrderedSet+MSIDExtensions.h"
```

Comment:
Why did all those headers get imported?

Reply:
I'm not sure if `MSIDConstants` isn't a good place for those but all categories defined in the project are not added to static library so I have to explicitly specify usage somewhere in the project.
Those should be in a file that is guaranteed to be used.

Reply:
I need to find a good place to have this workaround in IdentityCore. What's your recommendation?